sunpy/net/hek/utils.py

```diff
@@ -0,0 +1,282 @@
+
+    if attribute.get("is_point"):
+        coordinates = value.replace("POINT(", "").replace(")", "").split()
+        coord_list = [float(coordinate) for coordinate in coordinates]
+        coord_list[0] *= coord1_unit
+        coord_list[1] *= coord2_unit
+        if attribute["frame"] == "heliocentric":
+            center_sky = SkyCoord(coord_list[0], coord_list[1], [1]* len(coord_list) * u.AU, obstime=time, observer=observer, representation_type="cylindrical", frame=attribute["frame"])
+            return PolygonSkyRegion(vertices=center_sky)
+        else:
+            center_sky = SkyCoord(coord_list[0], coord_list[1], obstime=time, observer=observer, frame=attribute["frame"])
+        return PointSkyRegion(center=center_sky)
+    coordinates_str = value.split('((')[1].split('))')[0]
+    coord1_list = [float(coord.split()[0]) for coord in coordinates_str.split(',')] * coord1_unit
+    coord2_list = [float(coord.split()[1]) for coord in coordinates_str.split(',')] * coord2_unit
+    vertices = {}
+    if attribute["frame"] == "heliocentric":
+        vertices = SkyCoord(coord1_list, coord2_list, [1]* len(coord1_list) * u.AU, obstime=time, observer=observer, representation_type="cylindrical", frame="heliocentric")
+    else:
+        vertices = SkyCoord(coord1_list, coord2_list, obstime=time, observer=observer, frame=attribute["frame"])
+
+    return PolygonSkyRegion(vertices=vertices)
+
+def get_unit(unit):
+    """
+    Converts string into astropy unit.
+
+    Parameters
+    ----------
+    unit : str
+        The targeted unit
+
+    Returns
+    -------
+    unit : `~astropy.units.Unit`
+        Astropy unit object (e.g. :`~astropy.units.Unit` or :`~astropy.units.CompositeUnit`)
+
+    Raises
+    ------
+    ValueError
+        Because ``unit`` did not parse as unit.
+
+    Notes
+    ----
+    `A complete list of HEK parameters.
+    <https://www.lmsal.com/hek/VOEvent_Spec.html>`__
+    """
+    cm2 = u.def_unit("cm2", u.cm**2)
+    m2 = u.def_unit("m2", u.m**2)
+    m3 = u.def_unit("m3", u.m**3)
+    erg_per_cm3 = u.def_unit("ergs/cm^3", u.erg/u.ml)
+    hmi_pixel = u.def_unit("HMI pixel", (0.5*u.arcsec)/u.pixel)
+
+    aliases = {
+        "steradian": u.sr,
+        "arcseconds": u.arcsec,
+        "degrees": u.deg,
+        "sec": u.s,
+        "emx": u.Mx,
+        "amperes": u.A,
+        "ergs": u.erg,
+        "cubic centimeter": u.ml,
+        "square centimeter": cm2,
+        "cubic meter": m3,
+        "square meter": m2,
+        "ergs per cubic centimeter": erg_per_cm3,
+        "HMI pixels": hmi_pixel,
+        "radians": u.rad
+    }
+    with u.add_enabled_units([cm2, m2, m3]), u.set_enabled_aliases(aliases), warnings.catch_warnings():
+        # Units for coordinate frames have more than one unit, otherwise it will be just one unit.
+        # There is an assumption that coord1_unit, coord2_unit and coord3_unit are the same.
+        warnings.filterwarnings("ignore", category=u.UnitsWarning)
```

Comment:
What are these unit warnings?

Reply:
This is an example of the warnings that appear here:
```
WARNING: UnitsWarning: 'm/s/s' contains multiple slashes, which is discouraged by the FITS standard [astropy.units.format.generic]
2024-07-29 13:11:03 - astropy - WARNING: UnitsWarning: 'm/s/s' contains multiple slashes, which is discouraged by the FITS standard
```
We discussed that in a meeting before and we decided to ignore the warning because it's irrelevant to the HEK user.

Reply:
Ah yes, can we match the message? Just the "contains multiple slashes".

I want to try and be explicit to what messages we ignore.